### ISSUE TITLE
Show UI feedback when format / extension are incorrect

### ISSUE THREAD

**Issue by privatemaker** [sveltia/sveltia-cms#25]
I encountered the following issue whereby the config did not show an error (which makes sense) and the UI shows `This collection has no entries yet.` My collection settings were:

```
collections:
  - label: "Widgets"
    name: "widgets"
    folder: "content/widgets/"
    extension: "md"
    format: "yaml"
```

The UX bug (as I would call it) is that Sveltia did not return content in the UI *nor* inform the user as to *why* there was no content.

I assumed this is because Sveltia tried to parse the entire file as `yaml` but since it was a `.md + frontmatter` file, Sveltia failed silently. Converting the files `.yaml` and updating `extension: "yaml"` worked, but incorrect for my SSG.

Changing the value to `format: "frontmatter"` makes all the data correctly show, as expected 😄

**Suggestions**

1. Show user friendly message in UI 

```
## Oops

We found content, but were unable to display it.
Please make sure your settings and files are correct:

folder: "content/widgets/"
extension: "md"
format: yaml
```

2. Console log a parsing error (if there is?) such as this so there's a clue.

Relates to #17 but I think is discreet enough to warrant a separate issue and path to resolution.

**Comment by kyoshino** [comment]
Agreed! I’ll at least add a collection-level validation in the near future, including those format inconsistencies, duplicate collection names, etc.
